### ISSUE TITLE
Syntax Error in sfShader_createFromFile()

### ISSUE THREAD

**Issue by beley20** [BindBC/bindbc-sfml#4]
A comma is missing in graphics.d, line 568 after the geometryShaderFilename parameter. This causes a compilation error if sfmlSupport >= SFMLSupport.sfml240.

```
$ dub build
Performing "debug" build using dmd for x86_64.
bindbc-sfml 1.0.0: building configuration "static"...
../../.dub/packages/bindbc-sfml-1.0.0/bindbc-sfml/source/bindbc/sfml/graphics.d(568,118): Error: found `const` when expecting `)`
../../.dub/packages/bindbc-sfml-1.0.0/bindbc-sfml/source/bindbc/sfml/graphics.d(568,123): Error: semicolon expected following function declaration
../../.dub/packages/bindbc-sfml-1.0.0/bindbc-sfml/source/bindbc/sfml/graphics.d(568,123): Error: declaration expected, not `(`
dmd failed with exit code 1.
```

**Comment by mdparker** [comment]
Fixed in 1.0.1. Thanks!

**Comment by Ttimofeyka** [comment]
> Fixed in 1.0.1. Thanks!

Can you update this on DUB?

**Comment by mdparker** [comment]
That should happen automatically. Looks like something's off. I've triggered a manual update.
